server/middleware.ts

```diff
@@ -229,7 +236,7 @@ export function branchRedirector(requestType: ApplicationType): (request: expres
 				}
 			}
 			if (requestType === ApplicationType.Confirmation) {
-				if (request.params.branch !== user.confirmationBranch ) {
+				if (!user.accepted || branchName.toLowerCase() !== (user.confirmationBranch || "").toLowerCase()) {
```

Comment:
@petschekr `user.accepted` is not an indicator of if the user has been assigned a confirmation branch, it's better to check if user.confirmationBranch is defined.

Reply:
@ehsanmasdar Thanks for pointing that out. Does everything in 718422eaa51ad88f6ecdea1167376b5b11978328 make sense now with the distinction between confirmed and attending?